utils/build/declarations.d.ts

```diff
@@ -25,3 +25,4 @@ declare module '@zendeskgarden/css-menus';
 declare module '@zendeskgarden/css-forms';
 declare module '@zendeskgarden/css-arrows';
 declare module '@zendeskgarden/css-tabs';
```

Comment:
Side note: it looks like this got missed with the `react-tabs` CSS-in-JS.

Reply:
Removed.